src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/Licence.kt

```diff
@@ -13,15 +13,14 @@ data class Licence(
   @Schema(description = "Unique identifier for this licence within the service", example = "99999")
   val id: Long = -1,
 
-  @Schema(description = "The licence type code", allowableValues = ["AP", "PSS", "AP_PSS"], example = "AP")
+  @Schema(description = "The licence type code", example = "AP")
   val typeCode: LicenceType,
 
   @Schema(description = "The version number used for standard and additional conditions", example = "1.4")
   val version: String? = null,
 
   @Schema(
     description = "The current status code for this licence",
-    allowableValues = ["IN_PROGRESS", "SUBMITTED", "APPROVED", "REJECTED", "ACTIVE", "INACTIVE", "RECALLED"],
     example = "IN_PROGRESS",
```

Comment:
Do we want to take these off? Or add more? Or temporarily remove and then add them all back in when they're stable?